src/apify_client/consts.py

```diff
@@ -1,87 +1,54 @@
-    ACTOR_RUN_TIMED_OUT = 'ACTOR.RUN.TIMED_OUT'
-    #: The actor run was aborted
-    ACTOR_RUN_ABORTED = 'ACTOR.RUN.ABORTED'
-    #: The actor run was resurrected
-    ACTOR_RUN_RESURRECTED = 'ACTOR.RUN.RESURRECTED'
-
-    #: The actor build was created
-    ACTOR_BUILD_CREATED = 'ACTOR.BUILD.CREATED'
-    #: The actor build has succeeded
-    ACTOR_BUILD_SUCCEEDED = 'ACTOR.BUILD.SUCCEEDED'
-    #: The actor build has failed
-    ACTOR_BUILD_FAILED = 'ACTOR.BUILD.FAILED'
-    #: The actor build has timed out
-    ACTOR_BUILD_TIMED_OUT = 'ACTOR.BUILD.TIMED_OUT'
-    #: The actor build was aborted
-    ACTOR_BUILD_ABORTED = 'ACTOR.BUILD.ABORTED'
-
-
-class MetaOrigin(str, Enum):
-    """Possible origins for actor runs, i.e. how were the jobs started."""
-
-    #: Job started from Developer console in Source section of actor
-    DEVELOPMENT = 'DEVELOPMENT'
-    #: Job started from other place on the website (either console or task detail page)
-    WEB = 'WEB'
-    #: Job started through API
-    API = 'API'
-    #: Job started through Scheduler
-    SCHEDULER = 'SCHEDULER'
-    #: Job started through test actor page
-    TEST = 'TEST'
-    #: Job started by the webhook
-    WEBHOOK = 'WEBHOOK'
-    #: Job started by another actor run
-    ACTOR = 'ACTOR'
+import warnings
+from typing import Any
+
+from apify_shared.consts import BOOL_ENV_VARS as BOOL_ENV_VARS_
+from apify_shared.consts import BOOL_ENV_VARS_TYPE as BOOL_ENV_VARS_TYPE_
+from apify_shared.consts import DATETIME_ENV_VARS as DATETIME_ENV_VARS_
+from apify_shared.consts import DATETIME_ENV_VARS_TYPE as DATETIME_ENV_VARS_TYPE_
+from apify_shared.consts import FLOAT_ENV_VARS as FLOAT_ENV_VARS_
+from apify_shared.consts import FLOAT_ENV_VARS_TYPE as FLOAT_ENV_VARS_TYPE_
+from apify_shared.consts import INTEGER_ENV_VARS as INTEGER_ENV_VARS_
+from apify_shared.consts import INTEGER_ENV_VARS_TYPE as INTEGER_ENV_VARS_TYPE_
+from apify_shared.consts import STRING_ENV_VARS as STRING_ENV_VARS_
+from apify_shared.consts import STRING_ENV_VARS_TYPE as STRING_ENV_VARS_TYPE_
+from apify_shared.consts import ActorEventTypes as ActorEventTypes_
+from apify_shared.consts import ActorExitCodes as ActorExitCodes_
+from apify_shared.consts import ActorJobStatus as ActorJobStatus_
+from apify_shared.consts import ActorSourceType as ActorSourceType_
+from apify_shared.consts import ApifyEnvVars as ApifyEnvVars_
+from apify_shared.consts import MetaOrigin as MetaOrigin_
+from apify_shared.consts import WebhookEventType as WebhookEventType_
+
+DEPRECATED_NAMES = [
+    'BOOL_ENV_VARS',
+    'BOOL_ENV_VARS_TYPE',
+    'DATETIME_ENV_VARS',
+    'DATETIME_ENV_VARS_TYPE',
+    'FLOAT_ENV_VARS',
+    'FLOAT_ENV_VARS_TYPE',
+    'INTEGER_ENV_VARS',
+    'INTEGER_ENV_VARS_TYPE',
+    'STRING_ENV_VARS',
+    'STRING_ENV_VARS_TYPE',
+    'ActorEventTypes',
+    'ActorExitCodes',
+    'ActorJobStatus',
+    'ActorSourceType',
+    'ApifyEnvVars',
+    'MetaOrigin',
+    'WebhookEventType',
+]
```

Comment:
Maybe we could reexport only the four constants which were here before this PR, to keep it simpler.